packages/tools/client-debugger/client-debugger-view/src/components/AudienceView.tsx

```diff
@@ -62,6 +62,47 @@ export function AudienceView(props: AudienceViewProps): React.ReactElement {
 		};
 	}, [clientDebugger, audience, setAllAudienceMembers, setAudienceHistory]);
 
+	const audienceClientMetaData: AudienceClientMetaData[] = [];
+
+	for (const [clientId, client] of allAudienceMembers) {
```

Comment:
This is another case where we should be able to use the `map` method on the `Map` type.